docs/CLI.md

```diff
@@ -152,6 +152,8 @@ Alias: `-c`. The path to a Jest config file specifying how to find and execute t
 
 Alias: `--collectCoverage`. Indicates that test coverage information should be collected and reported in the output. Optionally pass `<boolean>` to override option set in configuration.
 
+Cannot be used together with `--v8-coverage`.
```

Comment:
How about extending `collectCoverage` to accept `boolean | 'v8'`? Would that be too much work? This would allow us to not introduce a new config and a runtime check for both flags being present at the same time, since that would be impossible state now

Reply:
Is overloading better?

Reply:
oooh, I like that idea way more!

Reply:
went with that